zhaquirks/konke/__init__.py

```diff
@@ -60,30 +76,13 @@ def handle_cluster_general_request(
             return
 
         attr = args[0][0]
-        if attr.attrid != 0x0000:
+
+        if attr.attrid != self.attributes_by_name["konke_button_event"].id:
             return
 
         value = attr.value.value
         event_args = {
-            PRESS_TYPE: self.PRESS_TYPES.get(value, value),
-            COMMAND_ID: value,
+            PRESS_TYPE: PRESS_TYPES[value],
+            COMMAND_ID: value.value,  # to maintain backwards compatibility
         }
         self.listener_event(ZHA_SEND_EVENT, event_args[PRESS_TYPE], event_args)
-
-    def deserialize(self, data):
```

Comment:
This `deserialize` override seems unnecessary. Are the unit tests accurate? They pass without it.

Reply:
Doesn't seem to be required (for my button model at least). Seems to work fine without it.